include/actuator/ActuatorBulkSimple.h

```diff
@@ -0,0 +1,107 @@
+// Copyright 2017 National Technology & Engineering Solutions of Sandia, LLC
+// (NTESS), National Renewable Energy Laboratory, University of Texas Austin,
+// Northwest Research Associates. Under the terms of Contract DE-NA0003525
+// with NTESS, the U.S. Government retains certain rights in this software.
+//
+// This software is released under the BSD 3-clause license. See LICENSE file
+// for more details.
+//
+
+#ifndef ACTUATORBULKSIMPLE_H_
+#define ACTUATORBULKSIMPLE_H_
+
+#include <actuator/ActuatorBulk.h>
+
+namespace sierra {
+namespace nalu {
+
+struct ActuatorMetaSimple : public ActuatorMeta
+{
+  ActuatorMetaSimple(const ActuatorMeta& actMeta);
+
+  // HOST ONLY
+  std::vector<std::string> turbineNames_;
+  std::vector<std::string> turbineOutputFileNames_;
+  bool filterLiftLineCorrection_;
+  bool isotropicGaussian_;
+  bool is_disk();
+
+  // TODO(SAKIEVICH) not certain all these need to be dual views
+  int maxNumPntsPerBlade_;
+  ActVectorDblDv epsilon_;
+  ActVectorDblDv epsilonChord_;
+  ActVectorDblDv epsilonTower_;
+  ActFixScalarBool useUniformAziSampling_;
+  ActFixScalarInt nPointsSwept_;
+  ActFixScalarInt nBlades_;
+
+  // Stuff for the simple blade
+  bool debug_output_;
+  std::size_t n_simpleblades_;
+  ActScalarIntDv  num_force_pts_blade_;
+  ActVectorDblDv  p1_;  // Start of blade
+  ActVectorDblDv  p2_;  // End of blade
+  ActVectorDblDv  p1zeroalphadir_;         // Directon of zero alpha at p1
+  ActVectorDblDv  chordnormaldir_;         // Direction normal to chord
+  ActVectorDblDv  spandir_;                // Direction in the span
+  // for the blade definitions
+  std::vector<std::vector<double>> chord_table_;
+  std::vector<std::vector<double>> twist_table_;
+  std::vector<std::vector<double>> elem_area_;
+  // for the polars
+  std::vector<std::vector<double>> aoa_polartable_;
+  std::vector<std::vector<double>> cl_polartable_;
+  std::vector<std::vector<double>> cd_polartable_;
+
+
+};
+
+struct ActuatorBulkSimple : public ActuatorBulk
+{
+  ActuatorBulkSimple(const ActuatorMetaSimple& actMeta, double naluTimeStep);
+
+  Kokkos::RangePolicy<Kokkos::DefaultHostExecutionSpace> local_range_policy();
+
+  void output_torque_info();
+  void init_epsilon(const ActuatorMetaSimple& actMeta);
+  virtual void zero_open_fast_views();
```

Comment:
you probably don't need this (plus it's gone in the new code)

Reply:
Torque calculation parameters have been deleted.